index.html

```diff
@@ -0,0 +1,423 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/367b368e67.js" crossorigin="anonymous"></script>
```

Comment:
Very good to use icon library, I like it :)

Reply:
Thanks !